test/decoders/test_video_decoder.py

```diff
@@ -831,6 +831,14 @@ def test_get_frames_by_pts_in_range_fails(self, device, seek_mode):
         with pytest.raises(ValueError, match="Invalid stop seconds"):
             frame = decoder.get_frames_played_in_range(0, 23)  # noqa
 
+    @pytest.mark.parametrize("device", cpu_and_cuda())
+    def test_get_key_frame_indices(self, device):
+        decoder = VideoDecoder(NASA_VIDEO.path, device=device, seek_mode="exact")
+        key_frame_indices = decoder._get_key_frame_indices()
+        size = key_frame_indices.size()
+        assert size[0] > 0
+        assert len(size) == 1
```

Comment:
@NicolasHug, is there a more PyTorch-y way to assert this?

Reply:
I can't find of an obvious one. In this specific case, maybe we can just hard-code `assert key_frame_indices.shape == (42,)`, but of course that's only working for this video.

On another note, I wonder if we should somewhat check the correctness of the returned values (still non-blocking)?